packages/reports/app/styles/navi-reports/components/navi-column-config/metric.less

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+
+@input-gray: #eef1f7;
```

Comment:
Since we're using this across multiple files now, can we move this to the core variables file?

Reply:
Oh it's actually not used in this file, just in `base.hbs`. Should I still add it to core or just leave it there?

Reply:
I think this color actually originates from a mock that Jon had so maybe it would be better to use a denali gray instead of this anyway.